config/initializers/remote_authenticatable.rb

```diff
@@ -13,8 +13,9 @@ def authenticate!
           begin
             if validate(resource)
               resource = resource.remote_authentication(auth_params)
-              if resource.access_token.nil?
-                populate_session_for_2fa(resource)
+              binding.pry
```

Comment:
I know it's removed in a later commit, but should we let this stay in for this commit?

Reply:
how did it get through 😠  I'll fix it

Reply:
it's gone now